android/build.gradle

```diff
@@ -25,19 +25,20 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 32
 
     sourceSets {
         main.java.srcDirs += 'src/main/kotlin'
     }
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 21
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
     lintOptions {
         disable 'InvalidPackage'
     }
     compileOptions {
+        coreLibraryDesugaringEnabled true
```

Comment:
hmm potestowaliście to dobrze z Proguardem i L8? My już dwa razy przechodziliśmy na desugaring i musieliśmy revertować z powrtoem do threeTenAbp ponieważ o ile rules proguardowe działały o tyle L8 pomimo wykluczenia desugarowanych klas (takie `j$.time.**` czy `j$.util.**`)  nadal powodował że apka czegoś nie znajdował w classpathie i crashe leciały

Reply:
Testy są w trakcie